public/locales/en/common.json

```diff
@@ -6,7 +6,12 @@
     "help": "Help",
     "welcome": "Welcome, name",
     "logout": "Log out",
-    "uio-home": "UI Online Home"
+    "uio-home": "UI Online Home",
+    "uio-home-url": {
+      "mobile": "https://uiom.edd.ca.gov/UIOM/Pages/Public/ExternalUser/UIOMobileLandingPage.aspx",
+      "desktop": "https://uio.edd.ca.gov/UIO/Pages/Public/ExternalUser/UIOnlineLandingPage.aspx",
+      "locale": "?l=en"
```

Comment:
It feels a bit weird for "?l=en" to be a value in a translations file, though not a big deal

Reply:
Is it common to put urls in the translation file? I was about to introduce a `public/links.json` file for this purpose, since you don't actually translate urls most of the time.

Reply:
I was being clever by putting the locale query here, as otherwise you have to import and interrogate the Next router object, whereas it will naturally find the right one if I keep it in the locale file. If it makes anyone too sus though happy to do it the other way.

For the URLs themselves, I don't know if I'd call it common? I was finding myself wanting the long string-y content out of my code, and I figured its not unusual to have distinct URLs by locale (though not in our case), so I tossed it here. I would be very onboard with a `public/links.json` file!